modules/programs/thefuck.nix

```
@@ -24,14 +24,23 @@ in
     enableNushellIntegration = lib.hm.shell.mkNushellIntegrationOption { inherit config; };
 
     enableZshIntegration = lib.hm.shell.mkZshIntegrationOption { inherit config; };
+
+    alias = lib.mkOption {
+      type = lib.types.str;
+      default = "fuck";
+      description = ''
+        `thefuck` needs an alias to be configured.
+        The default value is `fuck`, but you can use anything else as well.";
+      '';
```

```suggestion
      description = "Alias used to invoke `thefuck`.";
```

Default will already be documented by the option's default value.